### ISSUE TITLE
Parentheses unsupported?

### ISSUE THREAD

**Issue by chrisgulvik** [gregorysprenger/wf-ani#3]
### Description of the bug

Need to test more a clarify if parentheses are not supported. Especially for type strain comparisons it would be expected the assembly file to contain them such as "Leptospira_interrogans_(GCA_001292615.2).fna.gz"

This is an example error reported in the `errors.tsv` file for filenames containing parentheses.
``` 
877e682e483/.command.sh: line 28: syntax error near unexpected
```

**Comment by chrisgulvik** [comment]
Oops, complete error line example here:
PR051-R2_Leptospira_mtsangambouensis_	BLAST_ANI_BLAST_BIOPYTHON	/root/me/work/54/febc3b0babe06ef05e12f8f443fa0f/.command.sh: line 28: syntax error near unexpected token `('	/root/me/work/54/febc3b0babe06ef05e12f8f443fa0f

**Comment by gregorysprenger** [comment]
Fixed on the `dev` branch with commit 49c565e. Going to close this issue, but be aware it may be a little bit longer before it is moved to the `main` branch.
